x-pack/plugins/data_visualizer/public/application/index_data_visualizer/search_strategy/requests/esql_utils.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isPopulatedObject } from '@kbn/ml-is-populated-object';
+import { MAX_PERCENT, PERCENTILE_SPACING } from './constants';
+
+export interface ESQLQuery {
+  esql: string;
+}
+
+/**
+ * Helper function to escape special characters for field names used in ES|QL queries.
+ * https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-syntax.html#esql-identifiers
+ * @param str
+ * @returns "`str`"
+ **/
+export const escapeESQL = (str: string) => {
+  return `\`${str}\``;
+};
+
```

Comment:
Name of the function is misleading. It does not escape any characters, but simply wraps a string with ```. What happens if string contains ``` by itself?